app/components/appMainModule/examplePage1Module/testService1.js

```diff
@@ -57,7 +57,7 @@
         .then(getPageDataComplete)
         .catch(getPageDataFailed);
 
-      function getPageDataComplete(data, status, headers, config) {
+      function getPageDataComplete(data) {
```

Comment:
@KyleW we could probably delete this function here. It's referenced in line 85 for the factory to access, so it shouldn't cause any problems.